src/VisualStudio/Core/Def/Implementation/RQName/RQNameService.cs

```diff
@@ -1,25 +1,23 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServices.Implementation.RQName;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.RQName
+namespace Microsoft.VisualStudio.LanguageServices
 {
-    internal static class RQNameService
+    /// <summary>
+    /// Helpers related to <see cref="RQName"/>s.
```

Comment:
Should we explain what RQNames are here (and cref to IVsRefactorNotify)? It'd be nice if the Metadata view of IVsRefactorNotify included the remarks from the [msdn docs](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.shell.interop.ivsrefactornotify.aspx) with the grammar and examples.